### PR TITLE
[portaudio] add build script RCE

### PR DESCRIPTION
**Pull request by mcginty**
Via: https://github.com/RustAudio/rust-portaudio/issues/144

This is, as of yet, unfixed.

### CODE REVIEW

**Comment by Shnatsel** [comment]
Nice find. We'll need a linter for that.

File: crates/portaudio/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "portaudio"
+date = "2016-08-01"
```
Comment (tarcieri) [inline crates/portaudio/RUSTSEC-0000-0000.toml]: 2016?

Comment (tarcieri) [reply crates/portaudio/RUSTSEC-0000-0000.toml]: Oh wow, so it is...

**Comment by tarcieri** [comment]
@mcginty thanks for retroactively filing this! Wonder how many others like this there are

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2016-0003` in #105

**Pull request merged**